ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -1391,6 +1434,21 @@ tag =
           Left t
     _ -> Right $ tagChainSelReprocess t test
 
+  -- Tag this test case if we ever switched from a longer to a shorter chain in
```

Comment:
```suggestion
  -- Tag this test case if we ever switch from a longer to a shorter chain in
```

Reply:
Adopted in 93e5e45aa5e83c2e2ce2e09ebce66d81abb864d6